### PR TITLE
Draft 2: Use a minimal fixture for customer-specific data only

### PR DESCRIPTION
**Pull request by TheBestPessimist**
Here's a new approach to the 1 fixture to rule them all. This idea popped out after talking with @dh.

In our fixture we don't set the **WHOLE OBJECT** (and all the objects), but only the things that the customer wants to have changed and we use the existing fixtures and builders for everything else.

It is my understanding that the customer mostly wants familiar names (product, bpartner, pricelist, prod.category, etc.) and quantities, and the rest of the data should "mostly" be all right.

In this way the customer fixture is much smaller, without duplicated data (because I expect most stuff to be 80% the same between tests), and we'll also be able to use our builders and everything else just like we are doing now.

https://github.com/metasfresh/metasfresh-e2e/issues/286

### CODE REVIEW

File: cypress/integration/aaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaa.js [hunk 1]
```diff
@@ -56,6 +58,28 @@ let originalPriceList;
 let originalUOM;
 let originalTaxCategory;
 
+it('Read fixture and prepare the names', function() {
+  cy.fixture('a/a.json').then(f => {
+    priceSystemName = appendDate(f['priceSystemName'], date);
+    priceListName = appendDate(f['priceListName'], date);
```
Comment (cristinastefan90) [inline cypress/integration/aaaaaaaaaaaaaaaaaa/aaaaaaaaaaaaaa.js]: From my side it looks good.

File: cypress/integration/price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js [hunk 2]
```diff
@@ -56,7 +56,33 @@ let originalPriceList;
...
+    priceListSchemaName = appendHumanReadableNow(f['priceListSchemaName']);
+    surchargeAmount = f['surchargeAmount'];
+  });
+});
+
 describe('Create Price and Products', function() {
+  it('reaaaaaalyyyyyyyyy', function() {
+    cy.wait(5000);
+  });
+
```
Comment (cristinastefan90) [inline cypress/integration/price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js]: This looks a bit ugly:( do we reaaaalyyy need the wait in here?

Comment (TheBestPessimist) [reply cypress/integration/price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js]: Unfortunately, sometimes (eg. 2/5 maybe) we get that problem with `getLanguageSpecific`, so this wait is needed right now -- at least on my machine :(

Comment (cristinastefan90) [reply cypress/integration/price/add_a_product_to_a_pricelist_schema_and_create_a_new_PLV.js]: ok

**Pull request merged**